examples/static-hbase-encrypted/files/cloudwatch-install.tpl

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Instead of having the cloudwatch script copied in lots of places, could we have just one copy instead?

Reply:
Resolved.